### ISSUE TITLE
Some blocks are not generated properly, issue with setting default blockstate?

### ISSUE THREAD

**Issue by wormzjl** [McJty/LostCities#187]
I've tried to use the stairs from industrial renewal mod in city buildings, where I discovered all stairs pointing south (`industrialrenewal:catwalk_stair`) was instead pointing north (`industrialrenewal:catwalk_stair:2`). All other directions has no problems.

![image](https://user-images.githubusercontent.com/2989009/53398763-bbd47f80-39aa-11e9-8f6a-6b0c51b07eef.png)

The vanilla setblock command has no issues, so I wonder if there is a issue with default blockstate handling with lost cities?

For reference https://github.com/Cassiobsk8/Industrial_Renewal/issues/29

**Comment by McJty** [comment]
That's not the correct notation for meta: industrialrenewal:catwalk_stair:2
Should be industrialrenewal:catwalk_stair@2

**Comment by wormzjl** [comment]
> That's not the correct notation for meta: industrialrenewal:catwalk_stair:2
> Should be industrialrenewal:catwalk_stair@2

Yes I did use this @ notation in json files, I also noted that there are no issues with other directions

**Comment by wormzjl** [comment]
Also in the part definition, I've tried both `industrialrenewal:catwalk_stair` and `industrialrenewal:catwalk_stair@0`, both are facing north instead of south(as it should be).

**Comment by McJty** [comment]
Show me your palette json

**Comment by wormzjl** [comment]
Here is the part involved
https://gist.github.com/wormzjl/1388d25e8a18275a9f3328dddb744dee

I can send you the full config pack if you wish

**Comment by wormzjl** [comment]
Ok so this issue happens with vanilla blocks too.
I have some stairs which should be facing east (minecraft:brick_stairs)
But in generated structures they are actually facing north (minecraft:brick_stairs@3)
Updated title

Screenshot:
![image](https://user-images.githubusercontent.com/2989009/54533016-84456b80-4989-11e9-8d27-52cb616b9ac4.png)
